spec.md

```diff
@@ -2103,6 +2229,47 @@ Condition | gRPC Code | Description | Recovery Behavior
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on a node.
+
+`NodeExpandVolume` ONLY supports expansion of already node-published or node-staged volumes on the given `volume_path`.
+
+If plugin has `STAGE_UNSTAGE_VOLUME` node capability - `NodeExpandVolume` MUST be called after successful `NodeStageVolume`.
+`NodeExpandVolume` MAY be called before or after `NodePublishVolume` if plugin has `STAGE_UNSTAGE_VOLUME` node capability otherwise `NodeExpandVolume` MUST be called after successful `NodePublishVolume`.
+
+```protobuf
+message NodeExpandVolumeRequest {
```

Comment:
Would it be easier for plugin writers if we pass `VolumeCapability` to the `NodeExpandVolume` call, for a similar reason that we pass `capacity_range` in the call?

Reply:
I am not opposed to it. But why are you thinking of passing `VolumeCapability` here? Can you explain?

Reply:
Had an offline discussion with @gnufied. My argument is that `VolumeCapability` contains the filesystem that has been used to publish/stage the volume, and potentially other attributes or flags the plugin needs to know. But @gnufied made good points that:
1.  The plugin can find out the filesystem from the mount point (i.e., the target path), and the other attributes -- block/mount, access mode, and mount flags -- seems not required for now.
2. We can always add it later when needed, but if we add it now it would be a breaking change to remove it in the future.

So please drop this comment as well.